torch_em/model/unetr.py

```diff
@@ -27,7 +27,8 @@ def __init__(
         out_channels=1,
         use_sam_stats=False,
         use_mae_stats=False,
-        encoder_checkpoint_path=None
+        encoder_checkpoint_path=None,
+        final_activation=None,
```

Comment:
As discussed, it's not a good idea to hard-code the final activation to sigmoid. I updated this to use the same logic and syntax as in our U-Net implementation.